tests/formats/test_btunda.py

```diff
@@ -0,0 +1,31 @@
+import pytest
+from tests.test_lib import parse_build_compare_editor
+
+from mercury_engine_data_structures.formats.btunda import Btunda
+
+BTUNDA_PATH = "system/tunables/tunables.btunda"
+def test_btunda(dread_file_tree):
+    parse_build_compare_editor(Btunda, dread_file_tree, BTUNDA_PATH)
+
+def test_btunda_get_tunable(dread_file_tree):
+    btunda = dread_file_tree.get_parsed_asset(BTUNDA_PATH, type_hint=Btunda)
+
+    assert btunda.get_tunable(["ShotManager|CTunableShotManager", "rPlasmaBeamDiffusionExplosion", "uAmount"]) == 3
+
+def test_btunda_set_tunable(dread_file_tree):
+    btunda = dread_file_tree.get_parsed_asset(BTUNDA_PATH, type_hint=Btunda)
+
+    btunda.set_tunable(["SubAreaManager|CTunableSubAreaManager", "bKillPlayerOutsideScenario"], False)
+
+    assert btunda.raw.Root.hashTunables["SubAreaManager|CTunableSubAreaManager"]["bKillPlayerOutsideScenario"] is False
```

Comment:
Add an assertion the value had a different value before the set, otherwise we can't be sure the set worked.

Reply:
that value is True in both versions of the game so I don't really see why